proto/desmos/supply/v1/query.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+package desmos.supply.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v3/x/supply/types";
+
+
+// Query defines the gRPC querier service.
+service Query {
+
+  // TotalSupply queries the total supply of the given denom, converted with the given divider
+  rpc TotalSupply(QueryTotalSupplyRequest)
+      returns (QueryTotalSupplyResponse) {
```

Comment:
Instead of returning a `QueryTotalSupplyResponse`, is there any way we can simply return a string here instead? So the REST response instead of looking like this
```json
{
  "total_supply": "<amount>"
}
```

Can simply look like this:
```json
"<amount>"
```
Maybe something like [StringValue](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/wrappers.proto#L112) from `wrappers.proto` can work like this

Reply:
Done, but that makes the Protobuf linter angry...any way to skip to kind of checks on query returns types? @RiccardoM

Reply:
![Schermata 2022-04-05 alle 17 20 38](https://user-images.githubusercontent.com/26248202/161788235-ae5afdf2-a2ef-401a-b74a-ed5ac915bb2a.png)

Reply:
Considering CoinMarketCap requires the supply value to be returned simply as a value ([official example here](http://chainz.cryptoid.info/grs/api.dws?q=totalcoins)), we need a way to have this as well. 

Probably the best way would be to register a custom REST endpoint. This can be done using the `RegisterRESTRoutes` module method and registering a custom endpoint there.

Reply:
Both done 👍